x-pack/plugins/task_manager/server/lib/task_partitioner.test.ts

```diff
@@ -67,4 +76,19 @@ describe('getPartitions()', () => {
       247, 249, 250, 252, 253, 255,
     ]);
   });
+
+  test('correctly caches the partitions on 10 second interval ', async () => {
```

Comment:
Can we add a test for handling errors from the discovery service?

Reply:
Sure, resolved in this commit [e39827c](https://github.com/elastic/kibana/pull/189562/commits/e39827c923269686eef997b89d1917e8586fd8c4)